dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -122,7 +122,8 @@ internal KernelFunction(string name, string description, IReadOnlyList<KernelPar
 
         // Ensure arguments are initialized.
         arguments ??= new KernelArguments();
-        logger.LogFunctionInvokingWithArguments(this.Name, arguments);
+        logger.LogFunctionInvoking(this.Name);
+        logger.LogFunctionArguments(arguments);
```

Comment:
Do we have to split the logging here and for the results?

Reply:
Yes, because one is Information level, while the other is Trace level.